core/tracker/tracker.go

```diff
@@ -672,8 +672,8 @@ func newParticipationReporter(peers []p2p.Peer) func(context.Context, core.Duty,
 	// Initialise participation metrics to 0 to avoid non-existent metrics issue on startup.
 	for _, duty := range core.AllDutyTypes() {
 		for _, peer := range peers {
-			participationCounter.WithLabelValues(duty.String(), peer.Name).Add(0)
-			participationGauge.WithLabelValues(duty.String(), peer.Name).Set(0)
```

Comment:
@dB2510 setting `participationGauge` to 0 on startup means all peers start a "not participation" which is incorrect since the node didn't have to participate yet, so it hasn't missed or failed participating.